components/Welcome/WelcomeContent.tsx

```diff
@@ -1,6 +1,16 @@
 import NextButton from '../../components/NextButton'
+import { ReactNode, FunctionComponent } from 'react'
+import type { HeadlineComponent, LinkProps } from '../Default'
+import type { LinkComponent } from '../NextButton'
 
-function WelcomeContent({ newContractPath, t, Headline, Link }) {
+type WelcomeContentProps = {
```

Comment:
Pq em alguns momentos estamos usando FunctionalComponent e em outros não? :eyes:

Reply:
Acredito que seria bom manter um padrão

Reply:
to subindo um commit que padroniza